### PR TITLE
[Minor] Advanced tokens for header component

### PR DESCRIPTION
**Pull request by marcialfps**
New advanced tokens in header component.

### CODE REVIEW

File: lib/src/common/variables.js [hunk 1]
```diff
@@ -214,21 +213,49 @@ export const componentTokens = {
...
-    hoverHamburguerColor: globalTokens.mediumGreyBlack,
-    overlayColor: globalTokens.softBlack,
+    customContentFontFamily: "",
+    customContentFontStyle: globalTokens.type_normal,
+    customContentFontColor: "",
+    customContentFontSize: "",
+    customContentFontWeight: globalTokens.type_regular,
+    dropdownBackgroundColor: globalTokens.transparent,
+    dropdownHoverBackgroundColor: globalTokens.transparent,
     focusColor: globalTokens.blue,
```
Comment (aweell) [inline lib/src/common/variables.js:223]: Where is the `focusColor` token applied?

Comment (marcialfps) [reply lib/src/common/variables.js:223]: `focusColor` is applied to the hamburguer button when is clicked or focused.

Comment (aweell) [reply lib/src/common/variables.js:223]: Should we call it then `hamburguerFocusColor` instead? :hamburger:

Comment (marcialfps) [reply lib/src/common/variables.js]: `focusColor` renamed to `hamburguerFocusColor` in last commit.

**Review by aweell — APPROVED** [review]
👍🏼

**Pull request merged**